sktime/forecasting/compose/_reduce.py

```diff
@@ -564,7 +564,9 @@ def _fit(self, y, X=None, fh=None):
             if isinstance(timepoints, (pd.DatetimeIndex, pd.PeriodIndex)):
                 if timepoints.freq is None:
                     raise ValueError(
-                        "Please set frequency for DatetimeIndex or " + "PeriodIndex."
+                        "Please set frequency for DatetimeIndex or PeriodIndex. You "
+                        + "can use set_freq_hier function from sktime.utils.datetime "
+                        + "for this purpose (will convert DatetimeIndex to  PeriodIndex"
```

Comment:
typo, should be: `"for this purpose (will convert DatetimeIndex to PeriodIndex)"`